tock/tock/templates/base.html

```diff
@@ -7,7 +7,7 @@
     <title>Tock</title>
 
     {% load staticfiles %}
-    {% if request.user.user_data.is_late %}
+    {% if request.user.is_authenticated and request.user.user_data.is_late %}
```

Comment:
I think a comment here mentioning that both checks are required might be prudent.